front/src/components/UI/ActionInput/AddProduct.tsx

```diff
@@ -0,0 +1,86 @@
+import axios from "axios";
+import {
+  ButtonStyled,
+  FormStyled,
+  LabelStyled,
+  InputStyled,
+} from "./AuctionInput.styled";
+import { isEmpty } from "../../App/SalesArea/SalesArea";
+import { useState } from "react";
+
+const AddProduct = () => {
+  const [formInputsValidity, setFormInputsValidity] = useState({
+    name: true,
+    description: true,
+    category: true,
+    amount: true,
+    price: true,
+  });
+
+  const addProductHandler = (event: any) => {
+    event.preventDefault();
+
+    const enteredName = event.target.elements.name.value;
```

Comment:
event niech już będzie any bo też nie mogę od ręki znaleźć jak to otypować, ale tutaj te consty już mogą mieć nadane typy przez ciebie